.github/workflows/typescript_lint.yaml

```diff
@@ -0,0 +1,53 @@
+name: PR Workflow
```

Comment:
The name of the workflow should be more descriptive 😄

Reply:
Yeah good catch : D 

I have improved this in my recent commit.